src/soundio/soundmanager.cpp

```diff
@@ -121,7 +121,7 @@ QList<SoundDevicePointer> SoundManager::getDeviceList(
         const QString& filterAPI, bool bOutputDevices, bool bInputDevices) const {
     //qDebug() << "SoundManager::getDeviceList";
 
-    if (filterAPI == "None") {
+    if (filterAPI == MIXXX_NOSOUNDAPI_STRING) {
```

Comment:
I think we need to keep here the programatic string "None"

Reply:
I'll use existing `SoundManagerConfig::kDefaultAPI` so that string is defined in one place only
https://github.com/mixxxdj/mixxx/blob/5fe525fe73450039ef4cef2067830705244aed26/src/soundio/soundmanagerconfig.cpp#L14